### PR TITLE
Added a preloader

### PR DESCRIPTION
**Pull request by cryptofox17**
## Description
- Added a Responsive Preloader
- [Issue 43](https://github.com/Praful932/Kitabe/issues/43)

## Affected Dependencies
NA

## How has this been tested?
Ran it on local server and ensured that it didn't fail the tests.
## GIF Before
before_preloader

## GIF After

after_preloader_phonesize

after_preloader

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/Praful932/Kitabe/blob/master/CONTRIBUTING.md).
- [x] My changes do not edit/add any unrequired files.
- [x] My changes are covered by tests.

### CODE REVIEW

**Review by Praful932 — CHANGES_REQUESTED** [review]
@cryptofox17 Add before after SS & GIFs

File: templates/mainapp/index.html [hunk 1]
```diff
@@ -7,6 +7,58 @@
...
+            <div class="rect1"><img src="{% static 'mainapp/png/book.svg' %}" alt="book"></div>
+            <div class="rect2"><img src="{% static 'mainapp/png/book.svg' %}" alt="book"></div>
+            <div class="rect3"><img src="{% static 'mainapp/png/book.svg' %}" alt="book"></div>
+            <div class="rect4"><img src="{% static 'mainapp/png/book.svg' %}" alt="book"></div>
+            <div class="rect5"><img src="{% static 'mainapp/png/book.svg' %}" alt="book"></div>
+        </div>
+        <div class="stand"> </div>
+    </div>
+</div>
+<script>
```
Comment (maneprajakta) [inline templates/mainapp/index.html:24]: Add the script [here](https://github.com/Praful932/Kitabe/blob/1d95220c26c9b245bf3e8b10e3d685722a0bdc4c/templates/mainapp/index.html#L418) we already have a script tag.

**Review by maneprajakta — CHANGES_REQUESTED** [review]

File: templates/mainapp/index.html (hunk 1 above)
Comment (cryptofox17) [reply templates/mainapp/index.html:24]: Ok I will

**Comment by cryptofox17** [comment]
@Praful932 @maneprajakta done with changes

**Comment by Praful932** [comment]
@cryptofox17 We have deployed your [PR](https://kitabe-pipeline-pr-99.herokuapp.com/), The preloader loads in later..

**Comment by cryptofox17** [comment]
@Praful932 I am not really sure why that's happening. The content of the site is loading first even though I added an event listener to the window object and ensure that preloader is displayed first and it has a z index of 99. 
On the local server it seems to be ok, I have also attached the gifs in the description.
Could you tell me where I could have made the mistake?

**Comment by Praful932** [comment]
@cryptofox17 Reach out in the discord project channel, also update your branch with the latest upstream master

**Comment by Praful932** [comment]
@cryptofox17 Any updates?

**Comment by cryptofox17** [comment]
@Praful932 yeah, I tried multiple approaches but I feel like the script tag gets executed much later so the preloader is displayed later. 

So I have initially made the preloader container visible and based on the cookies I set it's display to none. I'm not sure if that would work. 

But I will update the PR Tomorrow so you can review it

**Comment by cryptofox17** [comment]
@Praful932 i am not sure why but there are conflicts in my PR and my fork is also a lot of commits ahead. Should i make a new PR instead?

**Comment by Praful932** [comment]
> @Praful932 i am not sure why but there are conflicts in my PR and my fork is also a lot of commits ahead. Should i make a new PR instead?

If the issue persists, Yes please

**Comment by cryptofox17** [comment]
@Praful932 since this PR has a lot of conflicts I decided to make a new one

**Pull request closed without merging**